contracts/TokenVesting.sol

```diff
@@ -34,33 +34,33 @@ contract TokenVesting is Ownable {
     constructor (IERC20 token) public {
         _token = token;
         //test data
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 0, 1900000000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 0, 117990560);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 6, 117990560);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 0, 184000000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 6, 184000000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 12, 220110000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 18, 220110000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 24, 226780000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 6, 132000000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 12, 132000000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 18, 136000000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 6, 495000000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 12, 495000000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 18, 510000000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 0, 1200000000);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 12, 354836500);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 18, 354836500);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 24, 354836500);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 30, 354836500);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 36, 354836500);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 42, 354836500);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 12, 266666666);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 18, 266666666);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 24, 266666666);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 30, 266666666);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 36, 266666666);
-        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 42, 266666666);
+        addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 0, 1900000000000000000000000000);
```

Comment:
Can we do something for better readability?

`uint256 scale = 10 ** decimals`

and then 

```
addVesting(0x9fB29AAc15b9A4B7F17c3385939b007540f4d791, now + 0, 1900000000 * scale);
...
```

Reply:
@0xAshish

Reply:
`SCALING_FACTOR = 1000000000000000000`

Reply:
@jdkanani  Since it's a one-time thing, why do operations there and fewer errors !?
for now, I am adding `decimals` but it won't be applicable on each entry, due to some of the amounts in fractions.

Reply:
fractions? Which amount is in fraction?

Reply:
not in this one I was just telling in general. and I've updated PR with SCALING_FACTOR.